### DISCUSSION TITLE
A320 Sunblind

### DISCUSSION THREAD

**Discussion by MrMooBerlin** [flybywiresim/aircraft#3724] [Ideas]
Recently during the day it seems to get very bright in the cockpit within the sim. By any chance would a sun blind be able to work to be able to reduce the amount of light into the cockpit by the sun. Its already got a stored position but no use.

**Comment by MapleNAV-TCASFAULT** [comment]
movable sunblinds will be added when the exporter is working correctly and when someone actually animates them
